apps/svelte.dev/src/routes/packages/packages-meta.ts

```diff
@@ -0,0 +1,312 @@
+	},
+	adapter: {
+		prompt:
+			'SvelteKit-specific deployment adapters or hosting integration tools. Examples: platform adapters (Vercel, Netlify, etc.), serverless deployment helpers, hosting configuration utilities. Key indicator: primary purpose is facilitating deployment to specific environments. NOT general server utilities or build tools.'
+	},
+	tooling: {
+		prompt:
+			'Development tools for Svelte that operate outside runtime code. Examples: build plugins, code generators, debugging utilities, CLI tools, linters. Key indicator: used during development process but not part of runtime application code. NOT runtime libraries, components, or server integrations.'
+	},
+	seo: {
+		prompt:
+			'Tools for improving search engine optimization in Svelte applications. Examples: meta tag managers, structured data generators, sitemap creators, link analyzers. Key indicator: primarily concerned with search engine visibility. NOT general head management or routing unless specifically SEO-focused.'
+	},
+	auth: {
+		prompt:
+			'User authentication and authorization solutions for Svelte. Examples: login systems, OAuth implementations, JWT handlers, permission managers. Key indicator: primarily concerned with user identity and access control. NOT general API clients or state management unless specifically auth-focused.'
+	},
+	integration: {
+		prompt:
+			'Connectors between Svelte and external services or systems. Examples: database clients, third-party API wrappers, payment processor integrations. Key indicator: primary purpose is connecting Svelte apps with external platforms. NOT general UI components or utilities unless they specifically bridge to external services.'
+	},
+	testing: {
+		prompt:
+			"Specialized testing tools designed for Svelte applications. Examples: component testing libraries, Svelte-aware test runners, store mocking utilities. Key indicator: explicitly built for testing Svelte code with awareness of Svelte's features. NOT general JavaScript testing tools that aren't Svelte-specific."
+	},
+	miscellaneous: {
+		prompt:
+			"Svelte utilities, helpers, and tools that don't fit cleanly into other categories. Examples: state management solutions, custom stores, actions, directives, form validation, performance optimizers, DOM manipulation utilities, scroll utilities, focus managers. Key indicator: provides functionality that enhances Svelte development but doesn't align with existing categories."
+	}
+} satisfies Record<string, { prompt: string }>;
+
+const SV_ADD = {
+	packages: [
+		'drizzle-orm',
+		'eslint',
+		'lucia',
+		'mdsvex',
+		'@inlang/paraglide-js',
+		'playwright',
+		'prettier',
+		'storybook',
+		'tailwindcss',
+		'vitest'
+	],
```

Comment:
The list has 10 items, `sv` has 12 official add-ons, and there is 3 on the ui (lucia, mdsvex and paraglide)

Some add-ons has more than 1 dep, but maybe we can make sure that at least one is present ? (eg: eslint). Maybe it's ok to not have devtools-json also.

Reply:
Thanks for catching this! I've added the missing ones

The other missing one is https://svelte.dev/docs/cli/sveltekit-adapter, but not quite sure how to display it in this format. I guess it could be `@sveltejs/adapter-auto`? Don't know if we really need it though because people would already expect that as an integration and we have a whole SvelteKit adapters category